src/main/frontend/src/components/modal/requests/RequestForm.js

```diff
@@ -0,0 +1,289 @@
+import en from 'date-fns/locale/en-GB';
+import { Formik } from 'formik';
+import moment from 'moment';
+import React, { Component } from 'react';
+import DatePicker, { registerLocale } from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker-cssmodules.css';
+import 'react-datepicker/dist/react-datepicker.css';
+import { connect } from 'react-redux';
+import * as Yup from 'yup';
+
+registerLocale('en-GB', en);
+
+const validationSchema = Yup.object().shape({
+  reason: Yup.string().required(),
+  absence: Yup.object({
+    id: Yup.string().required()
+  })
+});
+
+class RequestForm extends Component {
+  state = {
+    startDate: new Date(),
```

Comment:
```suggestion
    startDate: moment(),
```

And then we don't need to init `moment` objects all the time. Look into doing this for other methods in this component as well.

Reply:
Up until version 1.8.0, this package was using Moment.js. Starting v2.0.0, we switched to using native Date objects to reduce the size of the package.

It throws an error when I try to pass moment date.